### ISSUE TITLE
stylish styles

### ISSUE THREAD

**Issue by FinnFrotscher** [jiahaog/nativefier#365]
Hi
I use nativefier for dropbox paper and want to add a [stylish skin](https://userstyles.org/styles/134756/dark-gray-dropbox-paper) 
how is this possible?

**Comment by ronjouch** [comment]
Hi @FinnFrotscher, thanks for the feedback.

Bundling Chrome extensions is not supported now (see https://github.com/jiahaog/nativefier/issues/207), but try saving locally the custom CSS and injecting it with option `--inject /path/to/your/custom-dropbox-theme.css`.

**Comment by marciofao** [comment]
After packing up an app, can I add some css by changing or adding some file to any place in the folder hierarchy?

**Comment by pf71fs** [comment]
I agree, would love to see extensions support so that Stylish can work
